unittests/abi_tests.cpp

```diff
@@ -2700,8 +2700,8 @@ BOOST_AUTO_TEST_CASE(abi_large_signature)
       BOOST_CHECK_LE( (stop - start).count(), 51*1000 );
       // only contains hex_data if it didn't hit the deadline
       if( check_data ) {
-         BOOST_CHECK( var.get_object().contains( "data" ) );
-         BOOST_CHECK( !var.get_object().contains( "hex_data" ) );
+         BOOST_CHECK( !var.get_object().contains( "data" ) );
```

Comment:
change comment at line 2701
// contains only hex_data if it didn't hit the deadline

Reply:
OK

Reply:
@brianjohnson5972  changed as your comment, please approval this PR.